lib/generators/react_on_rails/base_generator.rb

```diff
@@ -91,7 +91,8 @@ def add_yarn_dependencies
             babel-plugin-macros"
 
         puts "Adding CSS handlers"
-        run "yarn add css-loader css-minimizer-webpack-plugin mini-css-extract-plugin style-loader"
+        # css-minimizer-webpack-plugin v6.0.0 requires Node 18+
```

Comment:
This is 100% the wrong thing to do. We should not be doing things like this to fix testing...Instead, just take Node 16 out of the test matrix.

Reply:
@ahangarha you could actually make the version of `css-minimizer-webpack-plugin` installed conditional on the version of `node` that is being used.

Have the generator check the result of `node -v` & then install the latest version of `css-minimizer-webpack-plugin` compatible with that `node` version.

Reply:
Sounds better 👍🏾